src/test/java/edu/hm/hafner/edu/hm/hafner/analysis/edu/hm/hafner/analysis/assertions/IssueAssertions.java

```diff
@@ -0,0 +1,289 @@
+        return this;
+    }
+
+    /**
+     * Proof fingerprint.
+     * @param fingerprint = expected.
+     * @return assertion.
+     */
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssertions hasFingerprint(final String fingerprint) {
+        // check actual not null
+        isNotNull();
+        // check condition
+        propertyEqualsCheck(actual.getFingerprint(), fingerprint, "fingerprint");
+        // Return this for Fluent.
+        return this;
+    }
+
+    /**
+     * Proof hashCode.
+     * @param hashCode = expected.
+     * @return assertion.
+     */
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssertions hasHashCode(final int hashCode) {
+        // check actual not null
+        isNotNull();
+        // check condition
+        propertyEqualsCheck(actual.hashCode(), hashCode, "hashCode");
+        // Return this for Fluent.
+        return this;
+    }
+
+    /**
+     * Proof other hashCode.
+     * @param hashCode = expected.
+     * @return assertion.
+     */
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssertions hasNotHashCode(final int hashCode) {
+        // check actual not null
+        isNotNull();
+        // check condition
+        if (Objects.equals(actual.hashCode(), hashCode)) {
+            failWithMessage("Expected issue's hashCode to be <%s> but was <%s>", hashCode, actual.hashCode());
+        }
+        // Return this for Fluent.
+        return this;
+    }
+
+    /**
+     * Proof toString.
+     * @param issuesAsString = expected.
+     * @return assertion.
+     */
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssertions isString(final String issuesAsString) {
+        // check actual not null
+        isNotNull();
+        // check condition
+        propertyEqualsCheck(actual.toString(), issuesAsString, "toString");
+        // Return this for Fluent.
+        return this;
+    }
+
+    /**
+     * Proof if a id exits.
+     * @return assertion.
+     */
+    @SuppressWarnings("NonBooleanMethodNameMayNotStartWithQuestion")
+    public IssueAssertions hasAId() {
+        // check actual not null
+        isNotNull();
+        // check condition
+        if (actual.getId() == null) {
+            failWithMessage("Expected issue has a Id");
+        }
+        // Return this for Fluent.
+        return this;
+    }
+
+    /**
+     * Easy check a property and generate message.
+     * @param actualValue = actual.
+     * @param expected = expected.
+     * @param propertyName = name of property.
+     * @param <T> = generic.
+     */
+    private <T> void propertyEqualsCheck(final T actualValue, final T expected, final String propertyName){
+        if (!Objects.equals(actualValue, expected)) {
+            failWithMessage("Expected issue's "+propertyName+" to be <%s> but was <%s>", expected, actualValue);
```

Comment:
Der Standard ist:
`private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`